### DISCUSSION TITLE
[Bug]: Font for Moonlight Material keeps "resetting" to Inter Regular.

### DISCUSSION THREAD

**Discussion by dustin-le** [AtomMaterialUI/material-theme-issues#510] [Q&A]
### Describe the bug

Out of all of the themes, Moonlight Material, which is the only theme I use, keeps getting its font set to 13pt Inter Regular, compared to the standard 12pt Menlo that all other themes use. Even after I reset the values to be 12pt Menlo in the .icls file, it works for a short period of time and gets set back to 13pt Inter Regular whenever I open a new project.

This seems to only be happening to my setup, as I had someone download the Material Theme plugin fresh from the marketplace and the Moonlight Material theme was fine. I've tried reinstalling the plugin and everything, but the 13pt Inter Regular curse keeps coming back.

### To Reproduce

1. Choose Moonlight Material theme

### Expected behavior

Have the font and size consistent with the rest of the themes: 12pt Menlo.

### Material Theme Version

8.13.2

### IDE

IntelliJ IDEA

**Comment by mallowigi** [comment]
Check whether you have the setting "automatically reset color scheme" enabled

**Comment by dustin-le** [comment]
I do not.

![image](https://github.com/mallowigi/material-theme-issues/assets/46009532/accf4bfe-623b-4c31-b77e-060cc27ebd30)

**Comment by mallowigi** [comment]
And "Use Editor Default Font"?

![CleanShot 2024-05-08 at 10 50 56@2x](https://github.com/mallowigi/material-theme-issues/assets/5015756/b3cd4b39-ef46-4494-af58-7ba4032fdec0)

**Comment by dustin-le** [comment]
Sadly I don't have premium. But the configured Default font is the 12pt JetBrains Mono (which looks the same as Menlo). I've set the color scheme to use the default and not use a custom one here:

![image](https://github.com/mallowigi/material-theme-issues/assets/46009532/ebb3ccd3-61ea-44c0-800b-72bf80bd021c)

Yet every time I open a new project, this option seems to be reset every time.

**Comment by mallowigi** [comment]
Can you post your material_theme.xml file here? I'll tell you what to change

**Comment by dustin-le** [comment]
```
<application>
  <component name="MaterialThemeConfig">
    <option name="isContrastMode" value="true" />
    <option name="isWizardShown" value="true" />
    <option name="overrideAccentColor" value="false" />
    <option name="pristineConfig" value="false" />
    <option name="treeFontSizeEnabled" value="true" />
    <option name="tabFontSizeEnabled" value="true" />
    <option name="upperCaseButtons" value="false" />
    <option name="settingsSelectedTab" value="4" />
    <option name="accentColor" value="74a0f1" />
    <option name="selectedTheme" value="MOONLIGHT" />
    <option name="userId" value="-4d03c409:186518c894b:-8000" />
    <option name="version" value="8.13.2" />
  </component>
</application>
```

**Comment by mallowigi** [comment]
I'm afraid I don't see what the problem can be. Quick question: in your color scheme settings, do you see multiple instances of the Moonlight color scheme by some chance ?
